packages/amplify-provider-awscloudformation/src/__tests__/root-stack-builder/__snapshots__/root-stack-builder.test.ts.snap

```diff
@@ -100,10 +100,63 @@ Object {
         "BucketName": Object {
           "Ref": "DeploymentBucketName",
         },
+        "PublicAccessBlockConfiguration": Object {
+          "BlockPublicAcls": true,
+          "BlockPublicPolicy": true,
+          "IgnorePublicAcls": true,
+          "RestrictPublicBuckets": true,
+        },
       },
       "Type": "AWS::S3::Bucket",
       "UpdateReplacePolicy": "Retain",
     },
+    "DeploymentBucketBlockHTTP": Object {
+      "Properties": Object {
+        "Bucket": Object {
+          "Ref": "DeploymentBucketName",
+        },
+        "PolicyDocument": Object {
+          "Statement": Array [
+            Object {
+              "Action": "s3:*",
+              "Condition": Object {
+                "Bool": Object {
+                  "aws:SecureTransport": false,
+                },
+              },
+              "Effect": "Deny",
+              "Principal": "*",
```

Comment:
Will deployment work in this case ? The IAM role used to deploy the project would need write permissions.

Reply:
This only denies unsecured traffic to the bucket. Any connection using HTTPS goes through